### PR TITLE
[Wallet] Add support for social wallet (Safeguards) import 

### LINKED ISSUE #770: Wallet recovery flow should have a social recovery option
### Expected Behavior

Wallet recovery flow should have social recovery options.  Related: #768 

### Current Behavior

Wallet only provides user input backup key recovery.

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

Implement the latest import wallet flow designs and add support for social wallet (Safeguards) import.

### Other Changes

A bit of css cleanup which triggered some trivial snapshot updates.

### Tested

Did import flow with backup key
Did import flow with safeguard phrases (half backup keys)

### Related issues

- Fixes #770 

Required follow-up for i18n: #1402 

### Backwards compatibility

Yes

wa-import-wallet
wa-import-wallet-social

### CODE REVIEW

**Comment by jmrossy** [comment]
Not sure why codecov says this drops coverage by > 4%, that's def not correct.

File: packages/mobile/src/import/ImportWallet.test.tsx [hunk 1]
```diff
@@ -1,172 +1,44 @@
...
-} from 'src/import/ImportWallet'
+import ImportWallet, { ImportWallet as ImportWalletClass } from 'src/import/ImportWallet'
 import { createMockStore, getMockI18nProps } from 'test/utils'
+import { mockMnemonic } from 'test/values'
 
 jest.mock('src/geth/GethAwareButton', () => {
   return Button
 })
 
-const SPANISH_MNEMONIC = 'avance colmo poema momia cofre pata res verso secta cinco tubería yacer eterno observar ojo tabaco seta ruina bebé oral miembro gato suelo violín'.normalize(
```
Comment (jmrossy) [inline packages/mobile/src/import/ImportWallet.test.tsx:19]: moved to backup/utils.test.ts

File: packages/mobile/src/import/ImportWallet.tsx [hunk 2]
```diff
@@ -6,52 +6,48 @@ import { fontStyles } from '@celo/react-components/styles/fonts'
...
+import { importBackupPhrase } from 'src/import/actions'
 import { nuxNavigationOptions } from 'src/navigator/Headers'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
 import { RootState } from 'src/redux/reducers'
-import { getMoneyDisplayValue } from 'src/utils/formatting'
-
-// Because of a RN bug, we can't fully clean the text as the user types
-// https://github.com/facebook/react-native/issues/11068
-export const formatBackupPhraseOnEdit = (phrase: string) => phrase.replace(/\s+/gm, ' ')
```
Comment (jmrossy) [inline packages/mobile/src/import/ImportWallet.tsx:25]: moved to backup/utils.ts

File: packages/mobile/src/account/__snapshots__/EditProfile.test.tsx.snap [hunk 3]
```diff
@@ -50,20 +50,14 @@ exports[`renders the EditProfile Component 1`] = `
         placeholder="yourName"
         rejectResponderTermination={true}
         style={
-          Array [
```
Comment (jmrossy) [inline packages/mobile/src/account/__snapshots__/EditProfile.test.tsx.snap:53]: This PR has a bunch of  see snapshot updates exactly like this. I just noticed that our text input component had an unnecessary style array, which was merged together into one style. Feel free to ignore these

File: packages/mobile/src/import/ImportWalletSocial.tsx [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+    this.props.importBackupPhrase(fullPhrase, false)
+  }
+
+  render() {
+    const { phrase1, phrase2 } = this.state
+    const { t, isImportingWallet } = this.props
+
+    return (
+      <SafeAreaView style={styles.container}>
+        <>
```
Comment (jeanregisser) [inline packages/mobile/src/import/ImportWalletSocial.tsx]: Looks like this fragment could be removed.

**Review by jeanregisser — APPROVED** [review]
Nice work! 👍

**Pull request merged**